clearwater-cluster-manager.root/usr/share/clearwater/clearwater-cluster-manager/scripts/check_cluster_state.py

```diff
@@ -31,21 +30,16 @@ def describe_clusters():
     cluster_values = {subkey.key: subkey.value for subkey in result.leaves}
 
     local_site_info = ""
-    if sites != "" and local_site != sites and etcd_version != "2.2.5":
+    if sites != "" and local_site != sites:
         local_site_info = " in the local site (" + local_site + ")"
 
-    # Put the cluster name in alphabetical order
-    print "This script prints the status of the Cassandra, Chronos, and Memcached clusters{}.".format(local_site_info)
+    print "This script prints the status of the data store clusters{}.\n".format(local_site_info)
```

Comment:
Is chronos a data store?
If not, this print statement could be misleading.

Reply:
I'd say it is - it stores timers